### ISSUE TITLE
Avoid text-wrapping in plots list

### ISSUE THREAD

**Issue by shish** [guydavis/machinaris#508]
**Describe the request**
Looks like currently the "plot" column is truncated in code, guessing an appropriate width, but that guess doesn't always give good results - CSS has some features which I think would be nicer :)

As a proof-of-concept, I added a style to all TD elements:

```
TD {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  max-width: 20em;
}
```

(If doing this for real I guess it should only apply to the `plot` column, but it doesn't seem to harm any of the others. Somebody good at CSS could also probably make it more dynamic, so that small screens are very truncated and large screens show the full text.)

**Images**
Before:
<img width="612" alt="Screenshot 2021-12-20 at 15 44 07" src="https://user-images.githubusercontent.com/40659/146794229-c3ffc6b1-8d61-4967-b7e2-fd02ece8aa87.png">
After:
<img width="717" alt="Screenshot 2021-12-20 at 15 49 59" src="https://user-images.githubusercontent.com/40659/146794945-c56b0edc-370e-4cac-8776-112389b674f4.png">

**Comment by guydavis** [comment]
Great suggestion!  This change is now in the `:develop` stream of images, should be in next [release](https://github.com/guydavis/machinaris/wiki/Releases#release-streams). Thanks!

**Comment by guydavis** [comment]
Now available in [v0.6.8](https://github.com/guydavis/machinaris/releases/tag/0.6.8).  Thanks again!
